_shared/project/tox.ini

```diff
@@ -30,10 +30,13 @@ setenv =
     dev: NEW_RELIC_APP_NAME = {env:NEW_RELIC_APP_NAME:{{ cookiecutter.slug }}}
     dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
 {% if cookiecutter.get("postgres") == "yes" %}
+    dev: ALEMBIC_CONFIG = {env:ALEMBIC_CONFIG:via/migrations/alembic.ini}
```

Comment:
This is a handy trick that means you no longer need to pass `-c conf/alembic.ini` every time you run `alembic`. You can just do `tox -e dev --run-command "alembic upgrade head"` and so on